clang/test/SemaHLSL/BuiltIns/WaveReadLaneAt-errors.hlsl

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
```

Comment:
I'm always suspicious when I see `verify-ignore-unexpected` especially if it doesn't specify a specific type. What diagnostics is this ignoring?

Reply:
Good catch. There are no ignored diagnostics and it is not needed.